tests/osaka/eip7692_eof_v1/eip6206_jumpf/test_nonreturning_validation.py

```diff
@@ -208,3 +208,32 @@ def test_jumpf_in_nonreturning(eof_test: EOFTestFiller, first: bool, code_prefix
 def test_callf_to_nonreturning(eof_test: EOFTestFiller, container: Container):
     """Test EOF validation failing due to CALLF to non-returning section."""
     eof_test(container=container, expect_exception=EOFException.CALLF_TO_NON_RETURNING)
+
+
+@pytest.mark.parametrize(
+    "container",
+    [
+        Container(
+            name="returning_section_aborts_jumpf",
+            sections=[
+                Section.Code(code=Op.CALLF[1] + Op.STOP, max_stack_height=1),
+                Section.Code(
+                    code=Op.PUSH0 * 2 + Op.RJUMPI[4] + Op.POP + Op.JUMPF[2] + Op.RETF,
+                    code_outputs=1,
+                ),
+                Section.Code(
+                    code=Op.PUSH0 * 2 + Op.RJUMPI[1] + Op.RETF + Op.INVALID,
+                    code_inputs=0,
+                    code_outputs=1,
+                ),
+            ],
+        ),
+    ],
+    ids=lambda x: x.name,
+)
+def test_eof_valid(
```

Comment:
I don't like the name. In this file we have already many EOF validation tests. So maybe be more specific? Something like `test_mixed_nonreturning` or `test_returning_nonreturning_mixed`?

Also, if this is more about JUMPF than nonreturning, we can move it to `test_jumpf_validation.py`.
Or otherwise, you may place both variant (CALLF and JUMPF) here.

Reply:
I thought we'd be adding more cases to the valid/invalid lists, like it's done in the 4750 tests. Will change to more specific